ufo-front-end/src/components/Header/header.js

```diff
@@ -1,11 +1,32 @@
+import Login from "../Login/login"
+import Logout from "../Login/logout"
+import { useState, useEffect } from "react"; 
+
 const Header = () => {
+
+    const [user, setUser] = useState()
+
+    const getUser = async () => {
+        const response = await fetch("http://localhost:5000/user", {
+          method: "GET",
+          headers: { Authorization: `Bearer ${localStorage.getItem("MyToken")}` },
+        });
+        const data = response.json();
```

Comment:
What are you doing with this data variable? It seems to store the json repsonse and... does nothing? Did you mean to set it to the user object on line 7?

Reply:
before I pushed I started on the next step getting the user details but haven't finished it yet,